Makefile

```diff
@@ -268,3 +269,4 @@ env: $(LIBGIT2)
 	echo 'CGO_CFLAGS="$(CGO_CFLAGS)"' >> $(BUILD_DIR)/.env
 	echo 'CGO_LDFLAGS="$(CGO_LDFLAGS)"' >> $(BUILD_DIR)/.env
 	echo 'KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS)' >> $(BUILD_DIR)/.env
+	echo 'GIT_CONFIG_GLOBAL=/dev/null' >> $(BUILD_DIR)/.env
```

Comment:
The change on line `99` makes this redundant. So we could remove it from the `make env` target - unless there is another reason for it to be here?

Reply:
I added this for people (like me) who use "make env" to generate a set of env variables to add to their IDE test templates so that tests work properly there as well.